<ui version="4.0" >
 <class>QgsProjectPropertiesBase</class>
 <widget class="QDialog" name="QgsProjectPropertiesBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Project Properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="titleBox" >
         <property name="title" >
          <string>Project Title</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>11</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLineEdit" name="titleEdit" >
            <property name="toolTip" >
             <string>Descriptive project name</string>
            </property>
            <property name="text" >
             <string>Default project title</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="btnGrpPrecision" >
         <property name="title" >
          <string>Precision</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>11</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radAutomatic" >
            <property name="toolTip" >
             <string>Automatically sets the number of decimal places in the mouse position display</string>
            </property>
            <property name="whatsThis" >
             <string>The number of decimal places that are used when displaying the mouse position is automatically set to be enough so that moving the mouse by one pixel gives a change in the position display</string>
            </property>
            <property name="text" >
             <string>Automatic</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radManual" >
            <property name="toolTip" >
             <string>Sets the number of decimal places to use for the mouse position display</string>
            </property>
            <property name="whatsThis" >
             <string>Sets the number of decimal places to use for the mouse position display</string>
            </property>
            <property name="text" >
             <string>Manual</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxDP" >
            <property name="toolTip" >
             <string>The number of decimal places for the manual option</string>
            </property>
            <property name="whatsThis" >
             <string>The number of decimal places for the manual option</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>decimal places</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Map Appearance</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>11</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Selection Color:</string>
            </property>
            <property name="buddy" >
             <cstring>pbnSelectionColour</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QgsColorButton" name="pbnSelectionColour" >
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Background Color:</string>
            </property>
            <property name="buddy" >
             <cstring>pbnCanvasColor</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QgsColorButton" name="pbnCanvasColor" >
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QGroupBox" name="grpDigitizing" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Digitizing</string>
         </property>
         <widget class="QCheckBox" name="mEnableTopologicalEditingCheckBox" >
          <property name="geometry" >
           <rect>
            <x>11</x>
            <y>21</y>
            <width>371</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text" >
           <string>Enable topological editing</string>
          </property>
         </widget>
         <widget class="QPushButton" name="mSnappingOptionsPushButton" >
          <property name="geometry" >
           <rect>
            <x>11</x>
            <y>95</y>
            <width>194</width>
            <height>36</height>
           </rect>
          </property>
          <property name="text" >
           <string>Snapping options...</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="mAvoidIntersectionsCheckBox" >
          <property name="geometry" >
           <rect>
            <x>11</x>
            <y>58</y>
            <width>371</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text" >
           <string>Avoid intersections of new polygons</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="btnGrpMapUnits" >
         <property name="title" >
          <string>Map Units</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>11</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radMeters" >
            <property name="text" >
             <string>Meters</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radFeet" >
            <property name="text" >
             <string>Feet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radDecimalDegrees" >
            <property name="text" >
             <string>Decimal degrees</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2" >
      <attribute name="title" >
       <string>Projection</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="cbxProjectionEnabled" >
         <property name="text" >
          <string>Enable on the fly projection</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QgsProjectionSelector" native="1" name="projectionSelector" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelector</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>radMeters</tabstop>
  <tabstop>radFeet</tabstop>
  <tabstop>radDecimalDegrees</tabstop>
  <tabstop>radAutomatic</tabstop>
  <tabstop>radManual</tabstop>
  <tabstop>spinBoxDP</tabstop>
  <tabstop>pbnSelectionColour</tabstop>
  <tabstop>pbnCanvasColor</tabstop>
  <tabstop>cbxProjectionEnabled</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
